### PR TITLE
add space between on this and Name

### PR DESCRIPTION
**Pull request by vgezer**
No strings will need to be changed on transifex.

### CODE REVIEW

File: lib/backend/localusers.php [hunk 1]
```diff
@@ -98,8 +98,8 @@ public function getAddressBooksForUser(array $options = array()) {
     public function getAddressBook($addressBookId, array $options = array()) {
 	$addressbook = array(
 	    "id" => $addressBookId,
-	    "displayname" => self::$l10n->t('On this') . self::$defaults->getName(),
-	    "description" => self::$l10n->t('On this') . self::$defaults->getName(),
+	    "displayname" => self::$l10n->t('On this') . ' ' . self::$defaults->getName(),
+	    "description" => self::$l10n->t('On this') . ' ' . self::$defaults->getName(),
```
Comment (tanghus) [inline lib/backend/localusers.php]: It should probably be

`"displayname" => self::$l10n->t('On this %s',  array(self::$defaults->getName())`

@LEDfan ?

Comment (LEDfan) [inline lib/backend/localusers.php]: Yes I think so...

Comment (vgezer) [inline lib/backend/localusers.php]: @tanghus @LEDfan Nope. This gives only {displayName} instead. I tried that already :/

Comment (tanghus) [inline lib/backend/localusers.php]: Really? Weird. Have you tried 
`"displayname" => self::$l10n->t('On this %s', array((string)self::$defaults->getName())` ?
It's no biggie if not possible, but interpolating variables is preferred.

Comment (tanghus) [inline lib/backend/localusers.php]: Ah, now I know:

``` php
"displayname" => (string)self::$l10n->t('On this %s', array(self::$defaults->getName()),
```

Translation strings are objects that needs to be cast to string. That happens automatically if you concatenate a string to them.

Comment (vgezer) [inline lib/backend/localusers.php]: Thanks. Now pushed the changes again.

**Comment by tanghus** [comment]
Thanks :)

**Pull request merged**